app/bundles/LeadBundle/EventListener/SegmentLogReportSubscriber.php

```diff
@@ -93,8 +93,12 @@ public function onReportGenerate(ReportGeneratorEvent $event)
                 $qb->expr()->isNotNull('log_removed.date_added')
             )
         );
-        $qb->setParameter('dateFrom', $event->getOptions()['dateFrom']->format('Y-m-d H:i:s'));
-        $qb->setParameter('dateTo', $event->getOptions()['dateTo']->format('Y-m-d H:i:s'));
+
+        $dateFrom = $event->getOptions()['dateFrom'] ?? (new \DateTime('-30 days'));
+        $dateTo   = $event->getOptions()['dateTo'] ?? (new \DateTime());
```

Comment:
If we use a different timezone in php.ini, the DateTime object will return the time based on the timezone set in php.ini. To ensure that the time is in UTC, we need to either forcibly set it to UTC or use the toUtcString method provided by DateTimeHelper.